docs/opening-with-cps.md

```diff
@@ -0,0 +1,35 @@
+# What determines when a project is opened with the new project system?
+
+When opening a solution, there are guids stored in the .sln file that tell VS to find a project system registered with that Guid. For example, if the sln file has an entry like 
```

Comment:
Since GUID is an acronym my preference is to fully capitalize it everywhere it is used. I realize that might be a matter of taste so I'll settle for it being capitalized consistently throughout the document. :-)